### ISSUE TITLE
he_offline_deployment: true is not enough to allow for airgapped install

### ISSUE THREAD

**Issue by drokath** [oVirt/ovirt-ansible-collection#434]
##### SUMMARY
When trying to install the hosted_engine with ovirt.ovirt.hosted_engine_setup in an airgapped environnement, the definition of he_offline_deployment is not enough.

##### COMPONENT NAME
ovirt.ovirt.engine_setup

##### STEPS TO REPRODUCE

```yaml
- name: Setup ovirt hosted engine
  hosts: node1.test.local
  tags: engine
  vars:
    he_ansible_host_name: node1.test.local
    he_appliance_password:  # set password here
    he_admin_password: # set password here
    he_bridge_if: "enp1s0"
    he_fqdn: "engine.test.local"
    he_vcpus: 2
    he_vm_mac_addr: # set mac here
    he_domain_type: "gluster"
    he_storage_domain_addr: "node1.test.local"
    he_storage_domain_path: "/gluster_bricks/engine"
    he_network_test: ping
    he_offline_deployment: true # To disable package updates
    he_vm_ip_addr: "192.168.121.50"
    he_vm_ip_prefix: "24"
    he_dns_addr: "192.168.121.2"
    he_default_gateway: "192.168.121.254"
    he_vm_etc_hosts: true
  roles:
    - role: ovirt.ovirt.hosted_engine_setup
```

##### EXPECTED RESULTS
Install should be able to continue.

##### ACTUAL RESULTS

```paste below
TASK [ovirt.ovirt.engine_setup : Install oVirt Engine package] *********************************************************
task path: /usr/share/ansible/collections/ansible_collections/ovirt/ovirt/roles/engine_setup/tasks/install_packages.yml:2

The full traceback is:
  File "/tmp/ansible_dnf_payload_n3iqu740/ansible_dnf_payload.zip/ansible/modules/packaging/os/dnf.py", line 670, in _base
  File "/usr/lib/python3.6/site-packages/dnf/base.py", line 399, in fill_sack
    self._add_repo_to_sack(r)
  File "/usr/lib/python3.6/site-packages/dnf/base.py", line 139, in _add_repo_to_sack
    repo.load()
  File "/usr/lib/python3.6/site-packages/dnf/repo.py", line 580, in load
    raise dnf.exceptions.RepoError(str(e))
fatal: [localhost -> engine.test.local]: FAILED! => {
    "changed": false,
    "invocation": {
        "module_args": {
            "allow_downgrade": false,
            "autoremove": false,
            "bugfix": false,
            "conf_file": null,
            "disable_excludes": null,
            "disable_gpg_check": false,
            "disable_plugin": [],
            "disablerepo": [],
            "download_dir": null,
            "download_only": false,
            "enable_plugin": [],
            "enablerepo": [],
            "exclude": [],
            "install_repoquery": true,
            "install_weak_deps": true,
            "installroot": "/",
            "list": null,
            "lock_timeout": 30,
            "name": [
                "ovirt-engine"
            ],
            "releasever": null,
            "security": false,
            "skip_broken": false,
            "state": "present",
            "update_cache": false,
            "update_only": false,
            "validate_certs": true
        }
    },
    "msg": "Failed to download metadata for repo 'ovirt-4.4-centos-ceph-pacific': Cannot prepare internal mirrorlist: Curl error (6): Couldn't resolve host name for http://mirrorlist.centos.org/?release=8-stream&arch=x86_64&repo=storage-ceph-pacific [Could not resolve host: mirrorlist.centos.org]",
    "rc": 1,
    "results": []
}
```

**Comment by drokath** [comment]
ovirt_engine_setup_perform_upgrade should be set to false in accordance with he_offline_deployment

**Comment by drokath** [comment]
engine_setup/tasks/main.yml has already been updated with the following line:

```
when: not ovirt_engine_setup_perform_upgrade|bool and not ovirt_engine_setup_offline|bool
```

This issue should be retested with an up-to-date collection. (not available in ovirt-node 4.4.10)

**Comment by drokath** [comment]
Already fixed for next release
